### ISSUE TITLE
[positioner] v2 alpha TrayCenter is not in the correct location

### ISSUE THREAD

**Issue by seanaye** [tauri-apps/plugins-workspace#653]
In rust when calling `move_window` to the `Position::TrayCenter` the acutal position is off on MacOS 13.5.1

```rs
                let tray = tauri::tray::TrayIconBuilder::new()
                    .icon(icon)
                    .on_tray_icon_event(|tray, event| {
                        let app = tray.app_handle();
                        tauri_plugin_positioner::on_tray_event(app, &event);
                        match event.click_type {
                            ClickType::Left => {
                                if let Some(window) = app.get_window("main") {
                                    let _ = window.move_window(Position::TrayCenter);
```
<img width="1510" alt="Screenshot 2023-10-11 at 11 37 45 AM" src="https://github.com/tauri-apps/plugins-workspace/assets/24496822/8700baa3-3931-47f8-83e1-89bfc97ad7ff">

**Comment by FabianLars** [comment]
Hi, is this the same as this https://github.com/tauri-apps/plugins-workspace/issues/724 ? == Does it only occur if you have an external monitor connected?

**Comment by seanaye** [comment]
@FabianLars I don't believe it is quite the same issue. I compiled my app with 0 external monitors. Just the M1 Macbook screen and it is still in the incorrect postion. 
<img width="912" alt="Screenshot 2023-11-26 at 3 31 23 PM" src="https://github.com/tauri-apps/plugins-workspace/assets/24496822/d58a57df-d1ca-4256-b738-3ec2964676c6">

**Comment by seanaye** [comment]
this project is open source if you want a repro https://github.com/seanaye/boom
